R/truncate_triangles.R

```diff
@@ -10,9 +10,8 @@
 #'   time points of reference and columns representing the delays.
 #' @param n Integer indicating the number of retrospective
 #'   truncated triangles to be generated, always starting from the most
-#'   recent reference time. Default is to only generate truncated matrices
-#'   that have sufficient number of rows to generate a nowcast from, though
-#'   any number can be specified.
+#'   recent reference time. Default is to generate triangles for 50% of the
```

Comment:
This doesn"t look like it actually reflects the n setting.

Reply:
Why isn't this also defaulting to nrow?

Reply:
What do you mean by the n setting? 
It can't be nrow, it has to use at least max_delay + 1 rows to estimate the delay (thats why I based it off of ncol)

Reply:
I think I need to fix the description though it shouldnt have changed

Reply:
You say 50 is the default but it isn't the default its some combination of column and row.

nrow(the data frame) would be longer than that so I am not clear why it can't be?